docs/upgrade.md

```diff
@@ -20,7 +13,7 @@ TL;DR is
 kubectl edit -n fleet-local default-os-image
 ```
 ```yaml
-apiVersion: rancheros.cattle.io/v1
+apiVersion: elemental.cattle.io/v1
```

Comment:
the osImage below is still pointing to the os2 version :o

Reply:
well yes all this upgrade examples require a closer look and review, they could be easily really outdated. I'd suggest leaving this as it for now and fix it in #241

Another options is to remove most of it and leave it as a TODO or TBC.